DetoxSync/DetoxSync/Spies/NSTimer+DTXSpy.m

```diff
@@ -72,6 +72,9 @@ CFRunLoopTimerRef __detox_sync_CFRunLoopTimerCreate(CFAllocatorRef allocator, CF
 static CFRunLoopTimerRef (*__orig_CFRunLoopTimerCreateWithHandler)(CFAllocatorRef allocator, CFAbsoluteTime fireDate, CFTimeInterval interval, CFOptionFlags flags, CFIndex order, void (^block) (CFRunLoopTimerRef timer));
 CFRunLoopTimerRef __detox_sync_CFRunLoopTimerCreateWithHandler(CFAllocatorRef allocator, CFAbsoluteTime fireDate, CFTimeInterval interval, CFOptionFlags flags, CFIndex order, void (^block) (id timer))
 {
+	NSLog(@"🤔 CFRunLoopTimerCreateWithHandler");
+
+	// What is this doing? We don't seem to be creating a trampoline here - I feel like we should be?
 	return (__bridge_retained CFRunLoopTimerRef)[[NSTimer alloc] initWithFireDate:CFBridgingRelease(CFDateCreate(allocator, fireDate)) interval:interval repeats:interval > 0 block:block];
```

Comment:
I don't understand why we're swizzling `CFRunLoopTimerCreateWithHandler` without adding any tracking for these timers (as far as I can tell).

Unless `initWithFireDate` somehow ends up calling `CFRunLoopTimerCreate`?

Reply:
This happens a lot.
I don't understand what this is doing.

Reply:
> Unless `initWithFireDate` somehow ends up calling `CFRunLoopTimerCreate`

Probably, yes. This can be easily checked by putting a breakpoint on `__detox_sync_CFRunLoopTimerCreate`.
https://github.com/wix/DetoxSync/tree/master/ExampleApp